<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mpFind">
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mpAddAgent">
         <property name="text">
          <string>Добавить агента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mpChangeAgent">
         <property name="text">
          <string>Редактировать агента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mpRemove">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mpTake">
         <property name="text">
          <string>Передать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mpAgents"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
